README.md

```diff
@@ -1,120 +1,93 @@
+</h3>
 
-WinUI is a user interface layer that contains modern controls and styles for building Windows apps. </span> As the native UI layer in Windows it embodies <a href="https://www.microsoft.com/design/fluent/#/">Fluent Design</a>, giving each Windows app the polished feel that customers expect.
+WinUI is a user interface layer that contains modern controls and styles for building Windows apps. The current generation is WinUI 3, which ships as part of the [Windows App SDK](https://aka.ms/winappsdk).
 
-WinUI 2 is a library of controls that provides official native Microsoft UI controls and features for Windows [UWP apps](https://docs.microsoft.com/windows/uwp/index). WinUI 2 can be used in any Windows 10 UWP XAML app, or in a Xamarin.Forms app running on Windows 10 using [native view embedding](https://docs.microsoft.com/xamarin/xamarin-forms/platform/native-views).
+- **Modern UI:** WinUI embodies Fluent Design to enable intuitive, accessible, and powerful experiences and the latest user interface patterns.
+**Developers in control:** Use .NET with C# or C++ to write apps that run great on x86, x64 and ARM.
+- **Optimized for modern experiences & hardware:** Create highly-demanding yet performant experiences with WinUI that are optimized for modern hardware.
+- **Part of the Windows App SDK:** The [Windows App SDK](https://aka.ms/winappsdk) is a set of libraries, frameworks, components, and tools that you can use in your apps to access powerful Windows platform functionality from all kinds of apps on many versions of Windows. The Windows App SDK combines the powers of Win32 native applications alongside modern API usage techniques, so your apps light up everywhere your users are.
+- **Powers key experiences in Windows:** Windows experiences and apps are built with WinUI. Its rich control set and styles make it easy to develop high-quality experiences used by millions every day.
 
-WinUI 3 is the next generation of the WinUI framework. It dramatically expands WinUI into a full UX framework, making WinUI available for all Desktop Windows apps for use as the UI layer.
- 
-## WinUI Community Calls
+<p align="center">
+<img src="docs/images/winui_os.png" alt="Powers key experiences in Windows" width="480"/>
+</br>
+</p>
 
-The WinUI community call is your monthly opportunity to learn about native UX development for Windows with WinUI.
+## 📋 Getting started with WinUI
 
-In these calls we’ll discuss the WinUI [roadmap](docs/roadmap.md), our status and your feedback.
+For WinUI, your app's users must be on Windows 10 1809 - Build 17763 or newer (including Windows Insider Previews).
 
-You can watch them online here on YouTube at the [Windows Developer channel](https://www.youtube.com/channel/UCzLbHrU7U3cUDNQWWAqjceA).
+The full documentation of WinUI can be found on [Microsoft Learn](https://learn.microsoft.com/en-us/windows/apps/desktop/):
+- [Get started with WinUI](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here)
```

Comment:
I think the learn.microsoft.com links should all exclude the "en-us/" part of the path, so they will choose the developer's default UI language.